src/people.js

```diff
@@ -26,6 +27,30 @@
  * @returns {PeopleResponse}  - Processed information
 */
 
-function peopleInformation(people) { }
+function peopleInformation(people) {
+
+  let listTotalAge = people.map((p) => p.age);
+  let listTotalHeight = people.map((p) => p.height);
```

Comment:
Are these two maps necessary? Instead of doing two loops, you could have used one forEach and push to the corresponding lists

Reply:
Use const instead of let